packages/govuk-frontend-review/rollup.config.mjs

```diff
@@ -39,6 +40,8 @@ export default defineConfig(({ i: input }) => ({
    * Input plugins
    */
   plugins: [
-    resolve()
+    resolve({
+      modulePaths: resolver(paths.app).modules
+    })
```

Comment:
**question** How come Rollup needs a nudge to look up in the workspace's `node_modules` first before jumping up to the root one?

Reply:
Bit of a stretch goal. You can install `govuk-frontend@4` into the review app too 😉

A few of the paths will need replacing with globs (to avoid the dist or no-dist prefix)

It's one of the things I stopped working on the other day